packages/client/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+History
```

Comment:
client is maybe too generic ? frontend maybe ?

Reply:
Maybe gonna need a more definitive statement than this before I change it. :laughing:

Reply:
I was calling it the 'galaxy-web-client' I think?

Reply:
I'm ok with galaxy-web-client

Reply:
Thanks all!

Reply:
beautiful, a web client